Berksfile.lock

```diff
@@ -9,7 +9,7 @@ GRAPH
   apt (7.5.12)
   common_attrs (0.4.6)
   golang (1.7.2)
-  oauth2_proxy (0.2.5)
+  oauth2_proxy (0.2.7)
```

Comment:
should this be `0.2.8` to match `metadata.rb`?

Reply:
I wonder why berks doesn't automatically update